components/navigation/Navigation.js

```diff
@@ -0,0 +1,15 @@
+import styled from "styled-components"
+
+const StyledDiv = styled.div`
```

Comment:
you component is called Navigation, so I would assume that you want to return html to navigate between pages.
Please use the nav element and name the styled component accordingly.

```suggestion
const StyledNavigation = styled.nav`
```

Reply:
It's not even supposed to be there yet.